htdocs/compta/prelevement/class/bonprelevement.class.php

```diff
@@ -766,8 +766,8 @@ public function create($banque = 0, $agence = 0, $mode = 'real', $format = 'ALL'
 			$datetimeprev = $executiondate;
 		}
 
-		$month = strftime("%m", $datetimeprev);
-		$year = strftime("%Y", $datetimeprev);
+		$month = date("%m", $datetimeprev);
```

Comment:
date should not be used because result depends on php timezone setup. It is not clear. Can you use instead dol_print_date(dol_now() with 3rd param 'tzuserrel' or 'gmt' .